### ISSUE TITLE
Add display of input mode

### ISSUE THREAD

**Issue by justbruce** [sspanak/tt9#155]
Would it be possible to display a toast or other text to indicate the input mode?  

  When switching, I find myself wondering which mode I'm in a lot, and there is no indication of which IM we are in.  

It's a small thing, and can certainly be worked around, but it would be a nice enhancement

**Comment by sspanak** [comment]
Yes, it is possible. Currently, there is only indication when changing the language when in ABC mode, because even with the indicator it could ambiguous.

Let's put together the requirements:
- Add a "Input Mode Change Notifications" dropdown setting under "Appearance".
- Description is:
  - When the setting is "Off": "No notifications.".
  - When the setting is "Minimum (Default)": "Display a toast message only when changing the language in ABC mode." (current behavior).
  - When "Always": "Display a toast message when changing the language or the input mode."
